Makefile

```diff
@@ -2,6 +2,9 @@ PIP := python -m pip --disable-pip-version-check
 WEBPACK := NODE_ENV=production ./bin/yarn webpack
 YARN := ./bin/yarn
 
+# Currently, this is only required to install black via pre-commit.
+REQUIRED_PY3_VERSION := $(shell awk 'FNR == 2' .python-version)
```

Comment:
Wondering if it'd be safer to filter down to the first line that starts with `3.` instead? Either way this looks good

Reply:
Eh, `.python-version` won't really be modified often.